frontend/src/pages/team/Settings/Danger.vue

```diff
@@ -52,6 +52,7 @@ export default {
     },
     data () {
         return {
+            applicationList: {},
             applicationCount: -1,
```

Comment:
Given we have `applicationList.count` now I'd drop `applicationCount` or make it a computed prop based on `applicationList` to save the duplication